build.gradle

```diff
@@ -169,28 +169,13 @@ jreleaser {
       System.setProperty('jreleaser.gpg.secret.key', secretKey)
     }
 
-    def passphrase = System.getenv('GPG_PASSPHRASE')
-    if (passphrase) {
-      System.setProperty('jreleaser.gpg.passphrase', passphrase)
-    }
   }
   deploy {
     maven {
       mavenCentral {
         sonatype {
           active = 'ALWAYS'
           url = 'https://central.sonatype.com/api/v1/publisher'
-          def sonatype_user = System.getenv('JRELEASER_MAVENCENTRAL_USERNAME')
-          def sonatype_pass = System.getenv('JRELEASER_MAVENCENTRAL_PASSWORD')
-          if (sonatype_user) {
-            System.setProperty('jreleaser.mavencentral.username', sonatype_user)
-          }
-          if (sonatype_pass) {
-            System.setProperty('jreleaser.mavencentral.password', sonatype_pass)
-          }
-          if (!sonatype_user || !sonatype_pass) {
-            logger.warn('Sonatype credentials are missing. This may cause deployment issues.')
```

Comment:
Oh helpful user warning, how I will miss thee. With any luck, other errors will be "helpful enough"(tm)